python/tskit/drawing.py

```diff
@@ -485,19 +490,21 @@ def __init__(
         self.symbol_size = symbol_size
         self.drawing = self.setup_drawing(style)
         self.node_mutations = collections.defaultdict(list)
-        self.mutations_over_root = False
+        self.mutations_over_roots = False
+        nodes = set(tree.nodes())
         for site in tree.sites():
             for mutation in site.mutations:
-                self.node_mutations[mutation.node].append(mutation)
-                if tree.parent(mutation.node) == NULL:
-                    self.mutations_over_root = True
+                if mutation.node in nodes:
+                    self.node_mutations[mutation.node].append(mutation)
+                    if tree.parent(mutation.node) == NULL:
+                        self.mutations_over_roots = True
```

Comment:
We should probably raise a ValueError here if ``mutation.node not in nodes`` - better to fail noisily than to silently hide mutations

Reply:
True. Note that we don't hide them in the "tree sequence" view: they are present on the X axis:

<img width="189" alt="Screenshot 2021-03-03 at 12 39 30" src="https://user-images.githubusercontent.com/4699014/109807225-814af080-7c1d-11eb-9d20-869d6d7b14a1.png">

But that's a bit obscure, so raising an error (or at least a warning) seems reasonable.

Reply:
Thinking about this, I'm inclined to issue a warning instead of raising an error,. After all, we don't currently raise an error when e.g. there are nodes in the node table which are not part of any tree. So I don't think there currently an expectation that we *need* to show all the contents of the tables, is there?

Reply:
Sure, warning makes sense.